static/js/compose.js

```diff
@@ -696,6 +733,11 @@ function validate_stream_message() {
         }
     }
 
+    if (user_acknowledged_all_everyone === false) {
+        compose_error("Please remove @all / @everyone or acknowledge that you will be spamming everyone!");
```

Comment:
This string needs internationalization tags; I think we don't have a good way to do that (This issue applies to all use of `compose_error`), so probably we can not worry about this and deal with it whenever we fix the rest of the `compose_error` usage.